server/utils/forms/inputs/calendarDayInput.test.ts

```diff
@@ -45,6 +45,26 @@ describe(CalendarDayInput, () => {
       })
     })
 
+    it('returns an error when multiple fields are empty', async () => {
+      const request = TestUtils.createRequest({
+        'deadline-year': '',
+        'deadline-month': '',
+        'deadline-day': '12',
+      })
+
+      const result = await new CalendarDayInput(request, 'deadline', messages).validate()
+
+      expect(result.error).toEqual({
+        errors: [
+          {
+            errorSummaryLinkedField: 'deadline-month',
+            formFields: ['deadline-month'],
+            message: 'The deadline must include a month',
```

Comment:
if only month were filled in would the 'highest priority' be year?

Reply:
Yep, I think that's how it'd behave. I haven't explored it in too much detail in the tests because we know we need to revisit this properly and already have a Jira task for it (IC-706).